src/frame/js/web-messenger.jsx

```diff
@@ -179,7 +179,7 @@ export function init(props = {}) {
             const {config: {app: {status}}} = store.getState();
 
             if (status !== 'active') {
-                return Promise.resolve();
+                throw new Error('App is inactive.');
```

Comment:
Yeah it should reset the store state. I'm maybe thinking about adding a `isInitialized` check on each public method to be cleaner. Currently, it would let you call them and fail with `Unauthorized`.